massa-grpc/src/api.rs

```diff
@@ -275,6 +275,8 @@ pub(crate) fn get_largest_stakers(
     // Create the context for the response.
     let context = Some(grpc_api::LargestStakersContext {
         slot: Some(cur_slot.into()),
+        // IMPORTANT TODO: tmp value because testnet_24 and massa proto latest are not synced atm
+        in_downtime: false,
```

Comment:
what does this do ?

Reply:
Coming from the proto doc: "If the chain is in downtime, we will return the last snapshot before the downtime"

This is an API type update it has nothing to do with this PR, I just put a filler so we can use our changes.

Reply:
ok thanks :)